types/foundry-vtt-api/types/constants.d.ts

```diff
@@ -0,0 +1,459 @@
+/**
+ * The global CONSTANTS object
+ */
+declare const CONST: {
+	vtt: 'Foundry VTT';
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/constants.d.ts;
  Line: 5;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.